### PR TITLE
Added changelog for 3.24.1

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,30 @@
+3.24.1:
+	- Added logging CFEngine component related SELinux denials in cf-support
+	  (ENT-12137)
+	- Also ignore IPv6 interface info
+	  Agent now also ignores interfaces listed in ignore_interfaces.rx when
```
Comment (craigcomstock) [inline ChangeLog:5]:
```suggestion
	- Agent now also ignores interfaces listed in ignore_interfaces.rx when
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+	    permission during remote copy. Previously it would be created with
+	    default permissions.
+	  - The destination file (i.e., '<FILENAME>') is no longer deleted on
+	    backup during file copy.  Previously it would be renamed to
+	    '<FILENAME>.cfsaved', causing the original file to dissappear. Now an
+	    actual copy of the original file with the same permissions is created
+	    instead.
+	  As a result, there will no longer be a brief moment where the original
+	  file is inaccessible.
+	  Ticket: (ENT-11988)
```
Comment (craigcomstock) [inline ChangeLog:22]:
```suggestion
	  file is inaccessible. Ticket: (ENT-11988)
```

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,30 @@
...
+	- commands promises with exit codes not matching any
+	             _returncodes attributes from classes body now log and
+	             error message not just an info message
+	  (CFE-4429, ENT-12103)
```
Comment (craigcomstock) [inline ChangeLog:26]:
```suggestion
	- commands promises with exit codes not matching any
	   _returncodes attributes from classes body now log and
	   error message not just an info message (CFE-4429, ENT-12103)
```

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,26 @@
...
+	  looking for IPv6 interface info. Variables such as
+	  'default:sys.hardware_mac[<INTERFACE>]' will no longer be defined for
+	  ignored interfaces. Ticket: (ENT-11840)
+	- Atomic copy_from in files promise
+	  Changes to 'files' promise in 'copy_from' attribute:
+	  - The new file (i.e., '<FILENAME>.cfnew') is now created with correct
+	    permission during remote copy. Previously it would be created with
+	    default permissions.
+	  - The destination file (i.e., '<FILENAME>') is no longer deleted on
+	    backup during file copy.  Previously it would be renamed to
```
Comment (larsewi) [inline ChangeLog:14]:
```suggestion
	    backup during file copy. Previously it would be renamed to
```

**Review by larsewi — APPROVED** [review]

File: ChangeLog [hunk 5]
```diff
@@ -1,3 +1,26 @@
...
+	  - The new file (i.e., '<FILENAME>.cfnew') is now created with correct
+	    permission during remote copy. Previously it would be created with
+	    default permissions.
+	  - The destination file (i.e., '<FILENAME>') is no longer deleted on
+	    backup during file copy. Previously it would be renamed to
+	    '<FILENAME>.cfsaved', causing the original file to dissappear. Now an
+	    actual copy of the original file with the same permissions is created
+	    instead.
+	  As a result, there will no longer be a brief moment where the original
+	  file is inaccessible. Ticket: (ENT-11988)
```
Comment (nickanderson) [inline ChangeLog:19]:
```suggestion
	  file is inaccessible. (ENT-11988)
```

File: ChangeLog [hunk 6]
```diff
@@ -1,3 +1,26 @@
+3.24.1:
+	- Added logging CFEngine component related SELinux denials in cf-support
+	  (ENT-12137)
+	- Agent now also ignores interfaces listed in ignore_interfaces.rx when
+	  looking for IPv6 interface info. Variables such as
+	  'default:sys.hardware_mac[<INTERFACE>]' will no longer be defined for
+	  ignored interfaces. Ticket: (ENT-11840)
```
Comment (nickanderson) [inline ChangeLog:7]:
```suggestion
	  ignored interfaces. (ENT-11840)
```

**Review by nickanderson — APPROVED** [review]
Suggested changes to remove `Ticket: ` prefix

**Review by nickanderson — APPROVED** [review]

**Pull request merged**